go/pkg/clickhouse/query-parser/types.go

```diff
@@ -0,0 +1,27 @@
+package queryparser
+
+import (
+	clickhouse "github.com/AfterShip/clickhouse-sql-parser/parser"
+)
+
+// SecurityFilter represents a row-level security constraint
+type SecurityFilter struct {
+	Column        string   // Column name
+	AllowedValues []string // Values user is allowed to access
```

Comment:
nit: I know we have them in the docs, but having another copy here might be better.